package.json

```diff
@@ -16,10 +16,6 @@
     "url": "https://github.com/winjs/react-winjs"
   },
   "license": "MIT",
-  "peerDependencies": {
-    "react": "0.13.x",
-    "winjs": "4.4.x"
-  },
```

Comment:
Some libraries in the React ecosystem still use `peerDependencies`: `react-dom`, `react-motion`
Some don't: `redux`, `react-router`

But yeah, @luisrudge is right. Most users of this library will be Windows users, and they'll be on npm3.